aws/resource_aws_alb_target_group_test.go

```diff
@@ -484,6 +490,7 @@ func testAccAWSALBTargetGroupConfig_basic(targetGroupName string) string {
   vpc_id = "${aws_vpc.test.id}"
 
   deregistration_delay = 200
+  slow_start = 0
```

Comment:
We should not include the default value in all the test configurations. It should only be specified in either configurations that actually require the setting or if the default is overridden.